protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -584,6 +584,60 @@ definitions:
       - overwrite
       #- upsert_dedup # TODO chris: SCD Type 1 can be implemented later
       - append_dedup # SCD Type 1 & 2
+      - insert
```

Comment:
Given that these do not need to be returned in the spec, we could have another enum that does not depend on `DestinationSyncMode` to describe the different operation modes for Data Activation